django_mongodb/expressions.py

```diff
@@ -66,6 +68,10 @@ def expression_wrapper(self, compiler, connection):
     return self.expression.as_mql(compiler, connection)
 
 
+def extra_where(self, compiler, connection):  # noqa: ARG001
+    raise NotSupportedError("QuerySet extras are not supported on MongoDB.")
```

Comment:
Let's use the same wording as https://github.com/mongodb-labs/django-mongodb/blob/8b98b58be3fe4ec7af5389cf8a7d2c1baa765f38/django_mongodb/compiler.py#L361